### ISSUE TITLE
Error regarding Patreon JSON file

### ISSUE THREAD

**Issue by gamefreq0** [Ridanisaurus/EmendatusEnigmatica#170]
### Issue description

Upon startup of an instance of 1.16.5 Minecraft with Forge 36.2.39 and this mod installed, an error is output to the log as follows:
`[11Dec2022 21:26:16.694] [emendatusenigmatica_supporter_downloader/INFO] [STDERR/]: [com.ridanisaurus.emendatusenigmatica.reward.PatreonSupporterRewardHandler:lambda$load$0:94]: com.google.gson.JsonSyntaxException: java.lang.IllegalStateException: Expected BEGIN_ARRAY but was BEGIN_OBJECT at line 1 column 2 path $
[11Dec2022 21:26:16.695] [emendatusenigmatica_supporter_downloader/INFO] [STDERR/]: [com.ridanisaurus.emendatusenigmatica.reward.PatreonSupporterRewardHandler:lambda$load$0:94]: 	at com.google.gson.Gson.fromJson(Gson.java:899)
[11Dec2022 21:26:16.695] [emendatusenigmatica_supporter_downloader/INFO] [STDERR/]: [com.ridanisaurus.emendatusenigmatica.reward.PatreonSupporterRewardHandler:lambda$load$0:94]: 	at com.ridanisaurus.emendatusenigmatica.reward.PatreonSupporterRewardHandler.lambda$load$0(PatreonSupporterRewardHandler.java:87)
[11Dec2022 21:26:16.695] [emendatusenigmatica_supporter_downloader/INFO] [STDERR/]: [com.ridanisaurus.emendatusenigmatica.reward.PatreonSupporterRewardHandler:lambda$load$0:94]: 	at java.base/java.lang.Thread.run(Thread.java:829)
[11Dec2022 21:26:16.695] [emendatusenigmatica_supporter_downloader/INFO] [STDERR/]: [java.lang.Throwable:printStackTrace:639]: Caused by: java.lang.IllegalStateException: Expected BEGIN_ARRAY but was BEGIN_OBJECT at line 1 column 2 path $
[11Dec2022 21:26:16.695] [emendatusenigmatica_supporter_downloader/INFO] [STDERR/]: [java.lang.Throwable:printStackTrace:639]: 	at com.google.gson.stream.JsonReader.beginArray(JsonReader.java:350)
[11Dec2022 21:26:16.695] [emendatusenigmatica_supporter_downloader/INFO] [STDERR/]: [java.lang.Throwable:printStackTrace:639]: 	at com.google.gson.internal.bind.ArrayTypeAdapter.read(ArrayTypeAdapter.java:70)
[11Dec2022 21:26:16.695] [emendatusenigmatica_supporter_downloader/INFO] [STDERR/]: [java.lang.Throwable:printStackTrace:639]: 	at com.google.gson.Gson.fromJson(Gson.java:887)
[11Dec2022 21:26:16.695] [emendatusenigmatica_supporter_downloader/INFO] [STDERR/]: [java.lang.Throwable:printStackTrace:639]: 	... 2 more
`

### Steps to reproduce

Create an instance of MC 1.16.5 with Forge 36.2.39
Add Emendatus Enigmatica 1.2.15

### Minecraft version

1.16.x or earlier (No longer being developed)

### Forge version

36.2.39

### Emendatus Enigmatica version

Older

### Other relevant versions

Emendatus Enigmatica 1.2.15

**Comment by Ridanisaurus** [comment]
Please provide the complete Log file.
```
If a (crash)log is relevant for this issue, link it here: (It's almost always relevant)

No response
```

**Comment by gamefreq0** [comment]
[debug.log](https://github.com/Ridanisaurus/EmendatusEnigmatica/files/10221958/debug.log)
Let me know if you need anything further

**Comment by Ridanisaurus** [comment]
Fixed in 1.2.16
